### DISCUSSION TITLE
payload decoder for RAK 7205 example code

### DISCUSSION THREAD

**Discussion by theguate** [aws-samples/aws-iot-core-lorawan#20] [Ideas]
Requesting guidance to get instructions to deploy a rak7200 payload decoder.

**Comment by asvand** [comment]
Hi Javier, please follow the guidelines in https://github.com/aws-samples/aws-iot-core-lorawan/tree/main/transform_binary_payload#how-to-deploy-an-existing-nodejs-binary-decoder to add a decoder code. As far as I can see, there are several open source versions of decoder available in Node.js. I am happy to review the code and merge it into the repo. Thanks, Andrei
